src/bundles/config.js

```diff
@@ -91,9 +91,11 @@ function getURLFromAddress (name, config) {
   if (!config) return null
 
   try {
-    const address = Array.isArray(config.Addresses[name])
+    const address = (Array.isArray(config.Addresses[name])
       ? config.Addresses[name][0]
-      : config.Addresses[name]
+      : config.Addresses[name])
+      .replace(/\/0\.0\.0\.0\//, '/127.0.0.1/') // fix for https://github.com/ipfs/ipfs-webui/issues/1821
+      .replace(/\/::\//, '/::1/')
```

Comment:
should we take this chance to more gracefully handle the array of addresses if it exists? do we always want the first one, or do we want the local one if one is found?